client-proxy/app.env

```diff
@@ -0,0 +1,6 @@
+RETENTION_HOURS=24
+RETENTION_BYTES=104857600
```

Comment:
Should we prefix the OpenLineage proxy specific environment variables with `OPENLINEAGE_PROXY_*`?

Reply:
This is already inside openlineage context. Do we need the prefix? What do we distinguish it from?

Reply:
We wouldn't need the prefix, but my assumption is that these are environment variables? Really, my comment was related to avoiding collisions with other environment variables (with the same name) already set for other libs/applications.